src/main/java/com/danielagapov/spawn/Models/ChatMessage.java

```diff
@@ -0,0 +1,35 @@
+package com.danielagapov.spawn.Models;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.UUID;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class ChatMessage implements Serializable {
+        @Id
+        @GeneratedValue(strategy = GenerationType.AUTO)
+        private UUID id;
+
+        private Instant timestamp;
+
+        @ManyToOne
+        @JoinColumn(name = "user_id", nullable = false)
+        private User userSender;
+
+        @ManyToOne
+        @JoinColumn(name = "event_id", nullable = false)
+        private Event event;
+
+        @Column(length = 1000)
```

Comment:
Let's not limit the content in the database entity layer, if we don't have to, as a general rule-of-thumb. Leave this to the front-end side to determine. Please remove this `length` constraint.

For example, at work, we had a "Comments" field in one of our forms that geotechnical engineers use. The field was restricted to 1000 chars, but they wanted that expanded. It would've been easy if this was configured in the front-end, but since it was at the database level, it required creating a new column and moving the data.